packages/scripts/src/application/application.ts

```diff
@@ -189,33 +185,18 @@ export class Application {
             close: () => disposables.dispose(),
         };
     }
+
     /**
      * @deprecated this is here for backwards compatibility with the old engine
      */
     protected analyzeFeatures(featureDiscoveryRoot: string, extensions: string[], buildConditions: string[]) {
         return analyzeFeatures(fs, this.basePath, featureDiscoveryRoot, undefined, extensions, buildConditions);
     }
-    public async create({ featureName, templatesDir, featuresDir }: ICreateOptions = {}) {
-        if (!featureName) {
-            throw new Error('Feature name is mandatory');
-        }
-
-        const { config } = await this.getEngineConfig();
 
-        const targetPath = pathToFeaturesDirectory(fs, this.basePath, config?.featuresDirectory || featuresDir);
-        const featureDirNameTemplate = config?.featureFolderNameTemplate;
-        const userTemplatesDirPath = config?.featureTemplatesFolder || templatesDir;
-        const templatesDirPath = userTemplatesDirPath
-            ? fs.join(this.basePath, userTemplatesDirPath)
-            : builtinTemplatesPath;
-
-        generateFeature({
-            fs,
-            featureName,
-            targetPath,
-            templatesDirPath,
-            featureDirNameTemplate,
-        });
+    /** @deprecated use engine-cli instead (`engine generate FeatureName`) */
+    // eslint-disable-next-line @typescript-eslint/require-await
+    public async create(..._: unknown[]) {
+        throw new Error('Deprecated');
```

Comment:
remove this completely. we wanted to remove this package and no longer use its cli. no point in placeholder error messages

Reply:
OK, I will just remove it here.
There's also a possibility to remove `electron-commons` and `electron-host` I think; it can be done as a one major, maybe 🤔